package-lock.json

```diff
@@ -28912,7 +28913,7 @@
         "node": ">=14"
       },
       "peerDependencies": {
-        "@opentelemetry/api": ">=1.3.0 <1.10.0"
+        "@opentelemetry/api": ">=1.0.0 <1.10.0"
```

Comment:
that's very weird.

Reply:
I wonder if it takes the dependency range from the published `npm` package that does have the same version as the one we have locally?

Reply:
I'll manually re-sync `package-lock.json` in this PR to get it merged.

Reply:
ah, I see - we forgot to sync `package-lock.json` over at #4992 so it still updated this but it was out of sync. `1.0.0` for these packages is actually correct. :facepalm:

Reply:
so I can just merge this and it'll be all good again. :slightly_smiling_face: